mathesar_ui/src/components/cell/data-types/components/number/NumberCellInput.svelte

```diff
@@ -41,4 +41,5 @@
     parentValue = newChildValue;
   }}
   on:input
+  --input-element-text-align="right"
```

Comment:
I'm fine with using style-props but we should be mindful while doing this since this would alter the dom structure and add an additional div to the component. This would make the component no longer a phrasing content.